packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -526,7 +526,10 @@ class WebpackCLI implements IWebpackCLI {
         }
 
         // Allow to use `./path/to/webpack-dev-server.js` outside `node_modules`
-        if (dependency === WEBPACK_DEV_SERVER_PACKAGE && fs.existsSync(WEBPACK_PACKAGE)) {
+        if (
+          dependency === WEBPACK_DEV_SERVER_PACKAGE &&
+          fs.existsSync(WEBPACK_DEV_SERVER_PACKAGE)
+        ) {
           skipInstallation = true;
         }
```

Comment:
The current behavior is to check for the package and if it doesn't exist we ask users to install it.

<img width="778" alt="Screenshot 2023-05-06 at 8 17 12 AM" src="https://user-images.githubusercontent.com/46647141/236594618-ab6b25ae-ad19-4935-962c-35f2e069425e.png">

If we use `require.resolve()` it will throw an error and exit the process. Should we use it?

Reply:
Good question, and another good question why we used `fs.existsSync` here, can you look at git history?

Reply:
@alexander-akait added on this PR https://github.com/webpack/webpack-cli/pull/2907

Reply:
@snitin315 I think we can remove this check, if developer provided a bad path, node.js itself generate a good error, so I think it is extra unnecessary check

Reply:
Not sure why I was added it :smile: